{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "Data Risk Manager",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "2.0.6"
                        }
                    ]
                }
            ],
            "datePublic": "2020-09-21T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "IBM Data Risk Manager (iDNA) 2.0.6 could allow a remote authenticated attacker to upload arbitrary files, caused by the improper validation of file extensions. By sending a specially-crafted HTTP request, a remote attacker could exploit this vulnerability to upload a malicious file, which could allow the attacker to execute arbitrary code on the vulnerable system. IBM X-Force ID: 184979."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "HIGH",
                        "baseScore": 8.8,
                        "baseSeverity": "HIGH",
                        "confidentialityImpact": "HIGH",
                        "exploitCodeMaturity": "UNPROVEN",
                        "integrityImpact": "HIGH",
                        "privilegesRequired": "LOW",
                        "remediationLevel": "OFFICIAL_FIX",
                        "reportConfidence": "CONFIRMED",
                        "scope": "UNCHANGED",
                        "temporalScore": 7.7,
                        "temporalSeverity": "HIGH",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.0/I:H/UI:N/A:H/PR:L/S:U/AV:N/AC:L/C:H/E:U/RC:C/RL:O",
                        "version": "3.0"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Gain Access",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2020-09-22T13:55:45",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/6335281"
                },
                {
                    "name": "ibm-idna-cve20204620-file-upload (184979)",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_XF"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/184979"
                }
            ]
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2020-4620",
        "dateReserved": "2019-12-30T00:00:00",
        "dateUpdated": "2020-09-22T13:55:45",
        "state": "PUBLISHED",
        "datePublished": "2020-09-22T13:55:45.311882Z"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}
